File: files/000-099/039/CH-039.R
library(tidyverse)
library(readxl)

input = read_excel("files/CH-039 Transformation.xlsx", range = 'B2:E10', col_names = F)
test  = read_excel("files/CH-039 Transformation.xlsx", range = 'G1:G22')


result = input %>%
  pivot_longer(everything(), names_to = NULL) %>%
  arrange(value) %>%
  na.omit() %>%
  distinct()

identical(result$value, test$`Result - Unique Code`)
# [1] TRUE

File: files/000-099/039/CH-039.py
import pandas as pd

input = pd.read_excel("CH-039 Transformation.xlsx", sheet_name="Sheet1", usecols="B:E", nrows=9)
test  = pd.read_excel("CH-039 Transformation.xlsx", sheet_name="Sheet1", usecols="G",  nrows=22)

input = input.stack().reset_index(drop=True)
input = input.sort_values().drop_duplicates().reset_index(drop=True)
input = pd.DataFrame(input, columns=['result'])

print(input['result'].equals(test['Result - Unique Code'])) # True